functions/completed-checks/function.json

```diff
@@ -1,5 +1,5 @@
 {
-  "disabled": false,
+  "disabled": true,
```

Comment:
This is no longer used.  Disabling a function is now done using env vars. Delete.

Reply:
removed all